### ISSUE TITLE
Question : DAG

### ISSUE THREAD

**Issue by mingtop** [vlfeat/matconvnet#310]
Here is my question:
         1. I want multi-inputs(all layers has different size of feature map,but the loss layer connect to the different subnet) can Matconvnet  do this job?

**Comment by vedaldi** [comment]
Yes, as far as I understand your question. However, the architecture needs to do something to merge the different streams of information into compatible dimensions eventually, unless you want the subnets to train independently.

**Comment by mingtop** [comment]
Thank you very much .   I will be very  gratitude you IF you can give me some other's implication or parper's titles   Best wishes to matcovnet team来自我的华为手机-------- 原始邮件 --------主题：Re: [matconvnet] Question : DAG (#310)发件人：Andrea Vedaldi 收件人：vlfeat/matconvnet 抄送：Yan Ming Yes, as far as I understand your question. However, the architecture needs to do something to merge the different streams of information into compatible dimensions eventually, unless you want the subnets to train independently.
